src/utils/WidgetUtils.js

```diff
@@ -423,32 +391,6 @@ export default class WidgetUtils {
         app.eventId = eventId;
         app.name = app.name || app.type;
 
-        if (app.type === 'jitsi') {
-            console.log("Replacing Jitsi widget URL with local wrapper");
-            if (!app.data || !app.data.conferenceId) {
-                // Assumed to be a v1 widget: add a data object for visibility on the wrapper
-                // TODO: Remove this once mobile supports v2 widgets
-                console.log("Replacing v1 Jitsi widget with v2 equivalent");
```

Comment:
What happened with this v1 -> v2 transformation path? Is it no longer needed?

Reply:
It moved to https://github.com/matrix-org/matrix-react-sdk/pull/4319/files#diff-0e2cb0e79e0d376b964d6d11e52c4c45R563